src/main/java/com/safetynet/alerts/logic/updaters/ResultModel.java

```diff
@@ -0,0 +1,25 @@
+package com.safetynet.alerts.logic.updaters;
+
+import com.safetynet.alerts.presentation.model.SafetyAlertsModel;
+
+/**
+ * Object used to hold an updated SafetyAlertsModel along with a boolean confirming if an operation was successful
+ */
+public class ResultModel {
+
+    private SafetyAlertsModel model;
+    private Boolean bool;
+
+    public ResultModel(SafetyAlertsModel model, Boolean bool){
+        this.model = model;
+        this.bool = bool;
+    }
+
+    public SafetyAlertsModel getModel() {
+        return model;
+    }
+
+    public Boolean getBool() {
```

Comment:
what this bool represents on the result model? I would rename it to something more meaningfull

Reply:
boolean represents if the operation was successful - updated now